tests/x11/multi_users_dm.pm

```diff
@@ -38,11 +38,17 @@ sub ensure_graphical_target {
 }
 
 sub restart_x11 {
-    ensure_multi_user_target;
-    ensure_graphical_target;
+    my ($self, $setnologin) = @_;
+    record_soft_failure('Workaround for graphical errors on cirrus when restarting X11 -> performing reboot (see bsc#1063060)');
+    type_string "reboot\n";
+    my $timeout = get_var('UEFI') ? 140 : 80;
+    $self->wait_boot(bootloader_time => $timeout, nologin => $setnologin, reallynologin => $setnologin);
+    #ensure_multi_user_target;
+    #ensure_graphical_target;
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit